esphome/components/mitsubishi_itp/climate.py

```diff
@@ -175,17 +179,17 @@
         ),
         "defrost": (
             "Defrost",
-            binary_sensor.binary_sensor_schema(),
+            binary_sensor.binary_sensor_schema(icon="mdi:snowflake-melt"),
             binary_sensor.register_binary_sensor,
         ),
         "hot_adjust": (
             "Hot Adjust",
```

Comment:
I'd take this opportunity to rename this to "Preheating" too?

Reply:
I thought about it, but wasn't sure since it's just going away entirely soon.  You're probably right though, I'll change it.